components.html

```diff
@@ -0,0 +1,1381 @@
+            <use xlink:href="img/icons.svg#action-backup-database" />
+          </svg>
+        </button>
+
+        <!--Locked Action-->
+        <button class="flex-container action-icon btn-no-outline">
+          <p class="action-title-locked text-center">Action Title</p>
+          <svg class="icon-24">
+            <use xlink:href="img/icons.svg#action-locked" />
+          </svg>
+        </button>
+      </div>
+
+
+
+      <!--COMPONENT LABEL-->
+      <p class="component-label">Checkbox / Radio Button / Toggle Switch</p>
+      <!--COMPONENT LABEL-->
+
+
+
+      <!--Checkbox-->
+      <ul>
+        <li>
+          <input type="checkbox" class="checkbox-default" name="" value="">
+          <label>Checkbox label</label>
+        </li>
+      </ul>
+
+
+
+      <!--Radio Button-->
+      <ul>
+        <li>
+          <input type="radio" name="name" value="">
+          <label>Radio button label</label>
+        </li>
+      </ul>
+
+
+
+      <br />
+
+
+
+      <!--Toggle-->
+      <div class="switch_box box_1">
+        <input type="checkbox" class="switch_1">
+      </div>
+
+
+
+      <br />
+
+
+
+      <!--Slider Switch-->
+      <div class="slider-switch-bg">
+        <div class="slider-switch-active">
+          <button>
+            <div style="margin-left:4rem;" class="slider-switch"></div>
+          </button>
+        </div>
+      </div>
+
+
+
+      <!--COMPONENT LABEL-->
+      <p class="component-label">Textboxes</p>
+      <!--COMPONENT LABEL-->
+
+
+
+      <!--Textboxes-->
+      <input type="text" name="" class="text-black" placeholder="Placeholder text...">
+
+      <br /><br />
+
+      <input type="text" name="" class="text-black" placeholder="Readonly Placeholder text..." readonly>
+
+      <br /><br />
+
+      <textarea rows="3" name="" class="text-black" placeholder="Placeholder text..."></textarea>
+
+
+
+      <!--COMPONENT LABEL-->
+      <p class="component-label">Dropdown</p>
+      <!--COMPONENT LABEL-->
+
+
+
+      <!--Dropdown-->
+      <select>
+        <option value="">store.test1.lewis.brains.us</option>
+        <option value="">store.test2.lewis.brains.us</option>
+        <option value="">store.test3.lewis.brains.us</option>
+      </select>
+
+
+
+      <!--COMPONENT LABEL-->
+      <p class="component-label">Tooltip Overlays</p>
+      <!--COMPONENT LABEL-->
+
+
+
+      <!--Speech bubble (arrow left)-->
+      <div class="flex-container">
+        <div class="speech-shadow">
+          <div class="speech-bubble-left">
+            <p>Text goes here</p>
+          </div>
+        </div>
+      </div>
+
+
+
+      <br />
+
+
+
+      <!--Speech bubble (arrow top)-->
+      <div class="flex-container">
+        <div class="speech-shadow">
+          <div class="speech-bubble-top">
+            <p>Text goes here</p>
+          </div>
+        </div>
+      </div>
+
+
+
+      <br />
+
+
+
+      <!--Speech bubble (button tooltip)-->
+      <div class="flex-container">
```

Comment:
Different HTML elements could help here to reduce the 'div-itis' 😂

Reply:
@bradt Is there a general practice for this? Like particular elements to use?

Reply:
Personally, I don't think it matters at all for an app like this that is never going to be crawled by a search engine. If you look at the markup behind Gmail, it's all divs. If you're doing markup for our sites though, you definitely want to be using the right HTML5 elements for the right situations as Google's crawler rewards that. [Mozilla's HTML5 guide](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/HTML5) seems pretty good at a glance, but there might be a better one out there.